packages/gatsby-plugin-sharp/src/index.js

```diff
@@ -401,7 +410,7 @@ function queueImageResizing({ file, args = {}, reporter }) {
   }
 }
 
-async function notMemoizedbase64({ file, args = {}, reporter }) {
+async function base64({ file, args = {}, reporter, cache }) {
```

Comment:
One note on the cache here. We changed (in Gatsby 2.1.5 or so) how plugins get a cache instance. What will happen is that if a plugin, e.g. gatsby-transformer-remark, passes its cache instance to this function, these nodes will be cached at `.cache/caches/gatsby-transformer-remark` which is probably _ok_ but worth noting!

@pieh any ideas for how we'd improve this? Is it worth improving?

Reply:
Unless we allow importing cache manager from gatsby, I don't think there is anything that can be done here so `gatsby-plugin-sharp` use it's own cache instead of `gatsby-transformer-remark` or `gatsby-transformer-sharp` caches